README.md

```diff
@@ -0,0 +1,47 @@
+# EzPlatform I18N
```

Comment:
We usually spell it "eZ Platform" (see http://ezplatform.com). Also, it is usually spelled i18n (lowercase).